public/modules/custom/hel_tpm_update_reminder/hel_tpm_update_reminder.module

```diff
@@ -0,0 +1,48 @@
+<?php
+
+function hel_tpm_update_reminder_cron() {
+  $ago = time()-(60*60*24*30*6);
```

Comment:
Change the calculations (60*60*24*30*6) and (60*60*24*14) either to strtotime("-x days") or just plain timestamp.

For future development it would be better to have those values in their own variables.